Tests/Indicators/FractalAdaptiveMovingAverageTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using NUnit.Framework;
+using QuantConnect.Indicators;
+using QuantConnect.Data.Market;
+
+namespace QuantConnect.Tests.Indicators
+{
+    [TestFixture]
+    public class FractalAdaptiveMovingAverageTests
+    {
+        [Test]
+        public void ComputesCorrectly()
+        {
+            FractalAdaptiveMovingAverage frama = new FractalAdaptiveMovingAverage("", 6, 100);
+
+            decimal[] values = { 441m, 442m, 446m, 438m, 400m, 442m, 448m, 437m, 435m, 431m, 450m, 451m };
+            decimal[] expected = { 441m, 442m, 446m, 438m, 400m, 442m, 444.51m, 441.67m, 438.58m, 434.69m, 435.84m, 436.73m, 438m };
```

Comment:
It's not essential, but it does test different parameters so provides additional regression confidence. If you'd like to minimize the codebase it's a candidate for removal.